src/lib/devConsole.js

```diff
@@ -1015,6 +1015,53 @@ const getPublicEncryptionKey = async organizationCode => {
 	}
 };
 
+const downloadFilesSequentially = async (
```

Comment:
Is this function actually downloading files sequentially? I thought it was just making a call to SMS to obtain the list of presignedUrls (along with the total size of download)?

Reply:
This function only gets the url and totalSize from SMS. The actual download from each of those presignedUrl is carried out by this function [downloadFileContent: here](https://github.com/adobe/aio-cli-plugin-api-mesh/pull/165/files#diff-108e1a808ec7725e3fc76957b8781af29c39328eb19282559f2b846a52c32d32R167)

Reply:
Sure, can we rename this function to better reflect that?
Something like `getLogFileUrls` or `getPresignedUrls`?

Reply:
Sure. will do